doc/readme.html

```diff
@@ -183,7 +183,7 @@ <h3>Configuring and building the library</h3>
         <b>Note:</b></p>
     <ul>
         <li> <code>/path/to/install/dir</code> is the directory which deal.II should be installed into. This can be a directory in your home directory (e.g., <code>~/bin/deal.II</code>) or a directory such as <code>/usr/local</code> if you have root privileges.
-            Another option is to use something like <code>`pwd`/../installed/</code> (note the backticks). Make sure the installation directory is not the same as the location where you unpacked <tt>deal.II/</tt>.
+            Another option is to use something like <code>`pwd`/../installed/</code> (note the backticks). 
```

Comment:
Backticks are popular but hard to read and understand. I think it would be better to use the environment variable, i.e, `something like <code>$PWD/../installed/</code>`

See
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#tag_08

Reply:
This is unrelated to this pull request, but I agree that this would be a useful change.

Reply:
Good point: I should have read the full line change. Lets skip this.